Makefile

```diff
@@ -27,9 +27,12 @@ open-docs: docs/build/html/index.html
 
 # jenkins doesn't seem to cope well with the docker's networing magic:
 #   dnf and `docker pull` malfunctions -- timeouts, network errors
-container:
+container-image:
 	docker build --network host --tag=$(CONU_REPOSITORY) .
 
+container:
```

Comment:
What about build deps to `container-image`?

Reply:
do you want to rebuild every time you invoke the container?